pythran/tests/test_numpy_fft.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+from test_env import TestEnv
+import numpy
+from pythran.typing import NDArray
```

Comment:
you must also edit the .travis.yml file to have travis run these tests, but I can take care of that if you want